.github/workflows/build.yml

```diff
@@ -61,6 +65,22 @@ jobs:
             arch: aarch64
             supported: true
             fetch-upstream: "true"
+          - variant: aws-k8s-1.22
+            arch: x86_64
+            supported: true
+            fetch-upstream: "true"
+          - variant: aws-k8s-1.22
+            arch: aarch64
+            supported: true
+            fetch-upstream: "true"
```

Comment:
These are temporary fetching from upstream source until EKS-D 1.22 source tarball is finalized.